_haikus/2k22 _mega.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: hacktober
+author: Megha Vishwakarma
+---
+
+During Hacktober 2022
```

Comment:
```suggestion
During Hacktober
```

Reply:
Hello @ltagliaferri  , I have updated the changes , Thank you !